src/net.h

```diff
@@ -797,7 +797,7 @@ class CConnman
         m_onion_binds = connOptions.onion_binds;
     }
 
-    CConnman(uint64_t seed0, uint64_t seed1, CAddrMan& addrman, bool network_active = true);
+    CConnman(uint64_t seed0, uint64_t seed1, AddrMan& addrman, bool network_active = true);
```

Comment:
As we are dropping the `C` from `AddrMan` and `AddrInfo`, is that a convention that will be applied elsewhere in the codebase from now on too? 

My understanding was the `C` before these structures indicated they are a `class`. Are we deciding to drop that indication from now on? 

Just trying to understand the rationale of name changes in https://github.com/bitcoin/bitcoin/pull/22950/commits/dd8f7f250095e58bbf4cd4effb481b52143bd1ed and https://github.com/bitcoin/bitcoin/pull/22950/commits/375750387e35ed751d1f5ab48860bdec93977f64

Reply:
Since https://github.com/bitcoin/bitcoin/pull/10461 the `developer-notes.md` states "Do not prefix class names with `C`"

Reply:
@rajarshimaitra: See https://bitcoin.stackexchange.com/questions/106558/what-do-the-classes-chainstatemanager-cchainstate-cchain-and-blockmanager-do-i or the Core PR review club that links to. New classes shouldn't have C in front of them but old ones aren't being renamed.

Reply:
Oh ok.. Thanks for the clarification. 
Just to understand further, its ok for old ones to drop the `C` in later PRs if they are being touched right? Just like the case here with `AddrMan` and `AddrInfo`?

So eventually the goal is to remove "C for classes" everywhere?

Reply:
New code should follow the style described in the developer notes. If you're substantially changing a piece of code, it makes sense to also update it to follow that convention.